### DISCUSSION TITLE
How to understand the affection of TN and TS in Wöehler Curve?

### DISCUSSION THREAD

**Discussion by XiongYuZeng** [boschresearch/pylife#75] [Q&A]
Hi Pylife group:
Need some help to understand TN and TS although I have read the help document about them. 
I realize that if I do not provide TN, TS, it will use the default value 1.  It seems that if TN and TS is 1, then I am not able to change the Wöehler curve to other failure probability. the code: woehler_curve_data.woehler.transform_to_failure_probability(new_failure_probability).to_pandas() will give me the same data no matter which value I assign to new_failure_probaility. Why?

In the help text, it stands beneath text, which I got few questions to ask.
--------------------------------------------------------------------------------
TN: The scatter in cycle direction, (N_90/N_10)
If the key is missing it is assumed to be 1.0 or calculated from TS if given.

TS: The scatter in cycle direction, (S_90/S_10)
If the key is missing it is assumed to be 1.0 or calculated from TN if given.
--------------------------------------------------------------------------------

Questions:
1. what does (N90/N_10) and (S_90/S_10) mean?
2. what is the reasonable value range of TS and TN I should use if I only have values for SD, ND, k_1 and k_2?

**Comment by johannes-mueller** [comment]
A Wöhler curve is always valid for a certain failure probability.  A 50% Wöhler curve means for example, that left and underneath it you'd expect less than 50% of the specimens to fail, right and above it it more than 50%.

So for every failure probability `P` there is a Wöhler curve which has its values `ND_P` and `SD_P`. The `TS` and `TN` values are the ratio of `N_90/N_10`, `TS` the ratio of `SD_90/SD_10`. So for example in [this picture](https://pylife.readthedocs.io/en/stable/demos/woehler_analyzer.html#Probit) the `TN` value is the distance between the 10% Wöhler curve and the 90% Wöhler curve in life time direction.

As for reasonable values, they tend to vary a lot between different material groups. 


## Edit

There is indeed one detail inaccurate in the documentation. The `TS` value is the ratio `SD_90/SD_10` whereas `TN` is the reatio between `N_90` and `N_10` (without the `D`). Where `N_xx` is the endurance limit in cycle direction for a certain load level above `SD_xx`

We will make the docs more accurate.

**Comment by johannes-mueller** [comment]
Please check #94
